doc/tutorials/transformer/masked_modelling.ipynb

```diff
@@ -0,0 +1,207 @@
+{
```

Comment:
<p>change to something like : "" Now we can upload the model into the BucketFS using the command-line. Unfortunately we cannot tell exactly when this process is completed. The Notebook's hourglass may not be a reliable indicator. The BucketFS will still be doing some work when the call issued by the notebook returns. Please wait for few moments longer before querying the model."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>We are not using the command line anymore.</p>

Reply:
<p>still there are some gramar issues remaining. "We need to load the model from the Huggingface hub into the BucketFS. This could potentially be a long process. <strong>Unfortunately,</strong> we cannot tell exactly when it has finished. <strong>The </strong>notebook's hourglass may not be a reliable indicator. BucketFS will still be doing some work when the call issued by the notebook returns. Please wait for <strong>a</strong> few moments after that, before querying the model."</p>